### PR TITLE
AmirHossein-Week2-Exercises and project

### PR DESCRIPTION
**Pull request by amirhshad**
By the way, Masoud and I are curious about this week merge while our files are not in folders by our name. Will it be okay if you merge both of this branch in the master?

### CODE REVIEW

File: Week2/js-exercises/ex2-whatsYourMondayWorth.js [hunk 1]
```diff
@@ -14,6 +14,14 @@
...
 function dayWorth(tasks, hourlyRate) {
   // put your code in here, the function does returns a euro formatted string
+  const totalTime = tasks
+  .map(item => item.duration)
+  .map(item => item/60)
+  .map(item => item * hourlyRate)
+  .reduce((a,b) => a + b,0)
+  return totalTime;
+  
+
```
Comment (n4cr) [inline Week2/js-exercises/ex2-whatsYourMondayWorth.js:24]: It says the output of the function should be a euro formatted string. this is not I believe

File: Week2/js-exercises/ex4-collectiveAge.js [hunk 2]
```diff
@@ -10,6 +10,9 @@
 
 function collectiveAge(people) {
   // return the sum of age for all the people
+  const ages = people.map(item => item.age)
+  const sumAges = ages.reduce((a,b) => a + b ,0)
```
Comment (n4cr) [inline Week2/js-exercises/ex4-collectiveAge.js:14]: You can only use a reduce and no map. `people.reduce((total, person)=> total + person.age, 0)`

**Comment by n4cr** [comment]
you have many merge conflicts.
Alwasy pull your code before pushing

File: Week2/js-exercises/ex4-collectiveAge.js (hunk 2 above)
Comment (amirhshad) [reply Week2/js-exercises/ex4-collectiveAge.js:14]: In the exercise it said use map to get the ages.

**Pull request closed without merging**